### PR TITLE
Docs: Improve Hacktoberfest Guidelines

### LINKED ISSUE #2229: [DOCS] Add Missing Preposition & Change in Guidelines
### Description

Hey EddieHub, I have found an issue in the **README.md** file!

In the below screenshot, we can see an arrow under the **A Note for Hacktoberfest Participants** section, located in README.md. This arrow suggests that the "Maintainers will opt-in these PRs" portion should be changed to "Maintainers will opt-in **to** these PRs" as the preposition "**to**" is missing from the portion.

Moreover, the line says, "Maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label." This needs to change as the project now contains the `hacktoberfest` topic, so the PRs will be automatically counted towards Hacktoberfest. However, profile update PRs that have the label `invalid` will not be counted towards Hacktoberfest.

### Screenshots

Missing-Preposition

### PR DESCRIPTION
**Pull request by Sriparno08**
## Fixes Issue

Closes #2229

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2324"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 > ### A note for Hacktoberfest participants:
 >
 > - Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT be counted for Hacktoberfest.
-> - Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values of the event will count. Maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
+> - Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values of the event will count. The project contains the `hacktoberfest` topic, so the PRs will be automatically counted towards Hacktoberfest. However, profile update PRs that have the label `invalid` will not be counted towards this event.
```
Comment (eddiejaoude) [inline README.md:11]: How does this sound

```suggestion
> - Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values of the event will count. The project contains the `hacktoberfest` topic, so the PRs will be automatically counted towards Hacktoberfest. However, profile update PRs will have the label `invalid` and therefore won't be counted towards this event.
```

**Comment by Sriparno08** [comment]
Sounds better, @eddiejaoude!

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Comment by Sriparno08** [comment]
You're welcome 😊

**Pull request merged**